src/libs/ajax.js

```diff
@@ -248,6 +263,13 @@ const User = signal => ({
     )
   },
 
+  acceptSamTos: async () => {
+    await fetchSam(
+      'register/user/v1/termsofservice',
+      _.mergeAll([authOpts(), { signal, method: 'POST' }, jsonBody('app.terra.bio/#terms-of-service')])
```

Comment:
Isaac raised this question on a past ToS PR and I don't know the answer and kept forgetting to bring it up so raising it here

> Should the URL come from the current location, since the app is deployed at multiple URLs?

Reply:
Sam only knows about this URL, so any other URLs sent in will be rejected. This is what was decided by legal and also it just makes things easier on the dev env side of things.